mantis-common-serde/build.gradle

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ext {
+    jacksonVersion = '2.10.+'
```

Comment:
Is this version definition needed? Doesn't seem to be used.

Reply:
Also I think the one in shaded (master branch) is moved to 2.12.+ IIRC.

Reply:
Thanks for catching this, I'll remove it.